File: JavaScript/8js 3/preFizz.js
/*
|--------------------------------------------------------------------------
| Pre-FizzBuzz Workout #1
|--------------------------------------------------------------------------
|
*/

function preFizz(n) {
   let res = [];
   for (let i = 1; i <= n; i++) {
      res.push(i);
   }
   return res;
}


function preFizz(n) {
   return [...Array(n)].map((_, i) => i + 1)
}


function preFizz(n) {
   return Array.from({
      length: n
   }, (_, i) => i + 1)
}


console.log(preFizz(1), [1]);
console.log(preFizz(2), [1, 2]);
console.log(preFizz(3), [1, 2, 3]);
console.log(preFizz(4), [1, 2, 3, 4]);
console.log(preFizz(5), [1, 2, 3, 4, 5]);

File: Python/8py 3/pre_fizz.py
"""
|--------------------------------------------------------------------------
| Pre-FizzBuzz Workout #1
|--------------------------------------------------------------------------
|
"""

def pre_fizz(n):
   return [i for i in range(1, n+1)]


def pre_fizz(n):
   return list(range(1, n+1))


def pre_fizz(n):
   return [i+1 for i in range(n)]


def pre_fizz(n):
   res = []
   for i in range(1, n+1):
      res.append(i)
   return res


print(pre_fizz(1), [1])
print(pre_fizz(2), [1,2])
print(pre_fizz(3), [1,2,3])
print(pre_fizz(4), [1,2,3,4])
print(pre_fizz(5), [1,2,3,4,5])